config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -1944,6 +1945,22 @@ def generate_presubmits_network_plugins():
                 optional=optional,
             )
         )
+        if plugin in supports_gce:
+            results.append(
+                presubmit_test(
+                    cloud='gce',
+                    k8s_version=k8s_version,
+                    kops_channel='alpha',
+                    name=f"pull-kops-e2e-gce-cni-{plugin}",
```

Comment:
```
	* found duplicate name after normalizing: (DashboardTab) e2ecalico
	* found duplicate name after normalizing: (DashboardTab) e2ecilium
	* found duplicate name after normalizing: (DashboardTab) e2ekindnet
	* found duplicate name after normalizing: (DashboardTab) e2ecalico
	* found duplicate name after normalizing: (DashboardTab) e2ecilium
	* found duplicate name after normalizing: (DashboardTab) e2ekindnet
	* found duplicate name after normalizing: (DashboardTab) e2ecalico
	* found duplicate name after normalizing: (DashboardTab) e2ecilium
	* ```

Reply:
the dashboard tab 🤔

Reply:
Thanks - fixing now.  Why we don't use the test name as the tab name I will never know...

Reply:
Gah, another conflict.  I'm just going to try using the test name as the tab name!  I can't think of a good reason not to...